backend/market/views.py

```diff
@@ -230,77 +237,75 @@ def destroy(self, request, *args, **kwargs):
 
 
 class Favorites(mixins.DestroyModelMixin, mixins.CreateModelMixin, viewsets.GenericViewSet):
```

Comment:
This is sus, we are not leveraging the viewset/serializer at all, put another pin here (or fix if you want this fix looks simpler)